js/web/lib/wasm/wasm-types.ts

```diff
@@ -247,7 +247,8 @@ export declare namespace JSEP {
       dataOffset: number,
       dataLength: number,
       builder: MLGraphBuilder,
-      desc: MLOperandDescriptor): MLOperand;
+      desc: MLOperandDescriptor,
+    ): MLOperand;
```

Comment:
Oh, I may dislike some of ORT's linting rules for the C++ code, but for TypeScript, I rather like this symmetrically putting the closing paren on the next line...

```
      builder: MLGraphBuilder,
      desc: MLOperandDescriptor,
    ): MLOperand;
```

Rather than jamming it onto the previous line as the last parameter:
```
      builder: MLGraphBuilder,
      desc: MLOperandDescriptor): MLOperand;
```

Tis clearer to read diffs when the last parameter is inserted/deleted/reordered.